CMakeLists.txt

```diff
@@ -77,7 +77,7 @@ option(
 set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 
 if (NOT MSVC)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wpedantic")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wpedantic -funwind-tables")
```

Comment:
why? afaik on x86 the unwind tables are there by default, no? is this required for arm? does this mean heaptrack won't work anymore for binaries compiled without unwind tables?

generally -> the backtrace backend must be configurable, I'm reluctant to remove libunwind. having the ability to switch is nice though, and would allow us to profile the different implementations too